plugins/inputs/ethtool/README.md

```diff
@@ -14,6 +14,21 @@ on the network device and driver.
   ## List of interfaces to ignore when pulling metrics.
   # interface_exclude = ["eth1"]
 
+  ## Support for reading statistics from interfaces in additional namespaces is
+  ## also supported, so long as the namespaces are named (have a symlink in
+  ## /var/run/netns). The telegraf process will also need the CAP_SYS_ADMIN
+  ## permission.
```

Comment:
Can I have you create a subsection under configuration section called 'namespaces', similar to what we have done in other plugins (e.g. [wireguard](https://github.com/influxdata/telegraf/blob/56eb914998b5008dfa0cdc55e7e983a8539c2f81/plugins/inputs/wireguard/README.md#error-operation-not-permitted) or [ipset](https://github.com/influxdata/telegraf/blob/56eb914998b5008dfa0cdc55e7e983a8539c2f81/plugins/inputs/ipset/README.md#using-systemd-capabilities)) for using sudo and explain why a user would need to do this and what the user might need to do.

Reply:
Subsection added.